### PR TITLE
Add support for time values with sub-second precision

### PR DESCRIPTION
**Pull request by 3rdcycle**
This fixes the issue that was reported on obspy-users on 8 June 2015.

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -29,6 +29,8 @@ master:
    * Upgrade to libmseed 2.16
  - obspy.io.shapefile:
    * New module for ESRI shapefile write support (see #1066)
+ - obspy.io.zmap
+   * Add support for time values with sub-second precision
```
Comment (krischer) [inline CHANGELOG.txt]: Can you add the issue number for future reference?

File: obspy/io/zmap/core.py [hunk 2]
```diff
@@ -137,6 +137,8 @@ def _serialize(self, catalog):
             origin = ev.preferred_origin()
             if origin:
                 dec_year = self._decimal_year(origin.time)
+                dec_second = origin.time.second + \
+                    origin.time.microsecond / float(1e6)
```
Comment (krischer) [inline obspy/io/zmap/core.py]: super minor nitpick: no need to wrap this in `float()` as the exponential notation will always produce floating point numbers.

**Comment by krischer** [comment]
Looks good to me. Thanks for that!

FYI for future bug fixes: can you please base them on the `releases` branch? Then they will also flow into the next minor release.  For this one we'll just cherry pick it onto the releases branch as the changes are so minor.

https://github.com/obspy/obspy/wiki/ObsPy-Git-Branching-Model

**Comment by 3rdcycle** [comment]
Ok, will use the releases branch next time. I don't think there's an issue number I could refer to since this has only been reported on the mailing list AFAIK.

**Comment by krischer** [comment]
This issue has a number: #1093 ;-)

**Comment by krischer** [comment]
Thanks. I'll take care that the changes also end up in the releases branch.

**Pull request merged**